### ISSUE TITLE
Error: Rule not found: terraform_dash_in_output_name

### ISSUE THREAD

**Issue by ashishmohite** [terraform-linters/tflint#599]
```hcl
config {
  module = true
  deep_check = true
  force = false

  aws_credentials = {
    profile                 = "my-dev"
    region                  = "us-east-1"
  }
}

rule terraform_dash_in_output_name {
  enabled = true
}
```
## Version
OS MacOS Catalina 10.15.2
TFLint v0.13.4
Terraform v0.12.3
+ provider.aws v2.29.0
+ provider.template v2.1.2

I am trying to add linting rules from https://github.com/terraform-linters/tflint/blob/master/docs/rules/README.md but when i add terraform_dash_in_output_name rule it gives me following error
```
Failed to check rule config. An error occurred:

Error: Rule not found: terraform_dash_in_output_name
```

**Comment by wata727** [comment]
terraform_dash_in_output_name will be added in v0.14. Please wait for the release of v0.14.
https://github.com/terraform-linters/tflint/milestone/8

**Comment by osulli** [comment]
@wata727 how do you feel about making atomic releases so new features can be delivered faster, instead of delaying proven commits/merges until a large bulk of work?

**Comment by wata727** [comment]
@osulli It's so good thing. Originally, v0.14 was planned to be released in the middle of the month, but because I was busy, everything was late.

I'm going to review the milestones and make adjustments to release them in this month.

**Comment by wata727** [comment]
v0.14.0 has been released
https://github.com/terraform-linters/tflint/releases/tag/v0.14.0

**Comment by eankeen** [comment]
i had this issue as well - thank you for the updates :)
